product_docs/docs/hadoop_data_adapter/2.1.0/05_installing_the_hadoop_data_adapter.mdx

```diff
@@ -8,71 +8,59 @@ The Hadoop Foreign Data Wrapper can be installed with an RPM package. During the
 
 <div id="using_an_rpm_package_to_install_the_data_adapter" class="registered_link"></div>
 
-## Installing the Hadoop Foreign Data Wrapper using an RPM Package
+## Installing the on Linux x86-64 (amd64)  
 
-You can install the Hadoop Foreign Data Wrapper using an RPM package on the following platforms:
+You can install the Hadoop Foreign Data Wrapper on the following operating systems on x86-64:
 
--   [RHEL or CentOS 7 PPCLE](#rhel_centos7_PPCLE)
--   [RHEL 7](#rhel7)
 -   [RHEL 8](#rhel8)
--   [CentOS 7](#centos7)
+-   [RHEL 7](#rhel7)
 -   [CentOS 8](#centos8)
+-   [CentOS 7](#centos7)
+-   [buntu/Debian](#on-debian-or-ubuntu)
```

Comment:
First 'U' is missing in name of the ubuntu platform [buntu/Debian]

Reply:
Jinx. We must have caught that error at the same time. Fixed.